plugins/warn_unused/warn_unused_main.ml

```diff
@@ -118,31 +118,33 @@ module Cmdline = struct
     end
   ]
 
-  let info = Term.info ~version ~man ~doc name
   let passes = [name; "--taint"; "--mark"; "--print"]
   let not_a_pass s = not (List.mem passes s)
 
   let pass name =
     let doc = sprintf "run $mname-%s pass" name in
-    Arg.(value & flag & info [name] ~doc)
-  let taint = pass "taint"
-  let print = pass "print"
-  let mark  = pass "mark"
+    Config.(flag name ~doc)
+
+  let taint_p = pass "taint"
+  let print_p = pass "print"
+  let mark_p  = pass "mark"
 
   let ignore_args _ _ _ = ()
-  let passes = Term.(pure ignore_args $taint $print $mark)
+  let passes {Config.get=(!)} = ignore_args !taint_p !print_p !mark_p
 
-  let parse () = match Term.eval ~argv (passes,info) with
-    | `Version | `Help -> exit 0
-    | `Error _ -> exit 1
-    | _ -> match Array.find argv ~f:not_a_pass with
+  let () =
+    let check_passes () = match Array.find argv ~f:not_a_pass with
       | None -> ()
-      | Some pass -> eprintf "Unknown pass: %s" pass; exit 1
-end
+      | Some pass -> eprintf "Unknown pass: %s" pass; exit 1 in
+    Config.manpage man;
+    Config.when_ready (fun {Config.get=(!)} ->
+        check_passes ();
```

Comment:
It was the artifact of a previous implementation that got carried over to this. I have removed it.